milatools/cli/vscode_utils.py

```diff
@@ -0,0 +1,37 @@
+import shutil
+import subprocess
+import sys
+from logging import getLogger as get_logger
+from pathlib import Path
+
+logger = get_logger(__name__)
+
+
+def running_inside_WSL() -> bool:
+    return sys.platform == "linux" and bool(shutil.which("powershell.exe"))
+
+
+def get_expected_vscode_settings_json_path() -> Path:
+    if sys.platform == "win32":
+        return Path.home() / "AppData\\Roaming\\Code\\User\\settings.json"
+    if sys.platform == "darwin":  # MacOS
+        return (
+            Path.home()
+            / "Library"
+            / "Application Support"
+            / "Code"
+            / "User"
+            / "settings.json"
+        )
+    if running_inside_WSL():
+        # Need to get the Windows Home directory, not the WSL one!
+        windows_username = subprocess.getoutput("powershell.exe '$env:UserName'")
+        return Path(
+            f"/mnt/c/Users/{windows_username}/AppData/Roaming/Code/User/settings.json"
+        )
+    # Linux:
+    return Path.home() / ".config/Code/User/settings.json"
+
+
+def vscode_installed() -> bool:
+    return bool(shutil.which("code"))
```

Comment:
The `$MILATOOLS_CODE_COMMAND` can be used to customize the command to use (e.g.  you could set it to `code-insiders`, if you're using that).

Reply:
Fixed: https://github.com/lebrice/milatools/blob/4be6efbddbb0216f8fa4f9a5b3e093cfd9b09c70/milatools/cli/vscode_utils.py#L38